### PR TITLE
Matchoffice base elixir image

### PR DESCRIPTION
**Pull request by juanpedrojose**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM matchoffice/rubies:2.3.1
```
Comment (skovsgaard) [inline Dockerfile:1]: Shouldn't we be pulling the trenpixster/elixir image up here? Or are we just using the Erlang Solutions package to provide Elixir?

Comment (juanpedrojose) [reply Dockerfile:1]: > we just using the Erlang Solutions package to provide Elixir

Correct, we are using the [officially documented](http://elixir-lang.org/install.html#unix-and-unix-like) way to install erlang and elixir

Comment (skovsgaard) [reply Dockerfile:1]: I wasn't questioning that way to do it. It's fine, really. I was just wondering if we could save some time by pulling a baser image for it, but it probably doesn't make sense at this scale.

**Review by skovsgaard — APPROVED** [review]
As long as it's settled that we're using the ESL repo, let's get this underway.

File: Dockerfile (hunk 1 above)
Comment (juanpedrojose) [reply Dockerfile:1]: I tried both ways - use base elixir and then install rubies, but I got to a point, that installing elixir to rubies is easier.

**Pull request merged**